frontend/public/components/pod.tsx

```diff
@@ -928,6 +928,7 @@ export const PodList: React.FC<PodListProps> = ({ showNamespaceOverride, showNod
       <VirtualizedTable<PodKind, PodRowData>
         {...props}
         aria-label={t('public~Pods')}
+        label={t('public~Pods')}
```

Comment:
@tvu20 kubernetes workload names do not get translated as they are considered universal names that don't need translation. You can see other virtualized tables like deployment, deployment-config, etc don't localize them.

Reply:
I added this because otherwise the StatusBox inside of the VirtualizedTable receives null as a label prop and therefore displays "Not found" instead of "No xx found" text.

Reply:
@tvu20 thanks for the explanation.